autoload/go/cmd.vim

```diff
@@ -2,6 +2,21 @@ if !exists("g:go_jump_to_error")
     let g:go_jump_to_error = 1
 endif
 
+if !exists("g:go_dispatch_enabled")
+    let g:go_dispatch_enabled = 0
+endif
+
+" Run either `make` or `Make` depending on the user's configuration
+" `Make` required the vim-dispatch plugin by Tim Pope.
+" https://github.com/tpope/vim-dispatch
+function! s:runMake(...)
```

Comment:
Can you make it that it accepts a boolean, such as `isSilent`. If it exists add `silent`, if not don't do it. Currently `:GoRun` is broken, because we do display the output of `:GoRun`